src/AccessibilityInsights.SharedUx/Controls/TestTabs/AutomatedChecksControl.xaml

```diff
@@ -125,7 +125,7 @@
                                 <GridViewColumn.CellTemplate>
                                     <DataTemplate>
                                         <StackPanel Orientation="Horizontal" MinWidth="200">
-                                            <fabric:FabricIconControl GlyphName="LadybugSolid" GlyphSize="Custom" FontSize="15" HorizontalAlignment="Center" VerticalAlignment="Center" Foreground="{DynamicResource ResourceKey=ButtonBlueBGBrush}" Margin="0,0,4,0" Visibility="{Binding Path=FileBugVisibility}"/>
+                                            <fabric:FabricIconControl GlyphName="LadybugSolid" GlyphSize="Custom" FontSize="15" HorizontalAlignment="Center" VerticalAlignment="Center" Foreground="{DynamicResource ResourceKey=ButtonBlueBGBrush}" Margin="0,0,4,0" Visibility="{Binding Path=FileBugVisibility}" ShowInControlView="False"/>
```

Comment:
![ladybug icon next to 'file issue'](https://user-images.githubusercontent.com/7775527/64988906-516d2900-d881-11e9-8565-4252bfaef0a0.png)